gitk-git/gitk

```diff
@@ -6678,6 +6703,9 @@ proc drawtags {id x xt y1} {
                    -font $font -tags [list tag.$id text]]
         if {$ntags >= 0} {
             $canv bind $t <1> $tagclick
+            if {$ntags_copy < $maxtags} {
```

Comment:
This seems to be necessary to get the the menu for up to `maxtags` (=3) tags including:
```
--- gitk
+++ gitk
@@ -6728,7 +6728,7 @@
                    -font $font -tags [list tag.$id text]]
         if {$ntags >= 0} {
             $canv bind $t <1> $tagclick
-            if {$ntags_copy < $maxtags} {
+            if {$ntags_copy <= $maxtags} {
               $canv bind $t $ctxbut [list tagmenu %X %Y $id $tag_quoted]
             }
         } elseif {$nheads >= 0} {

```

Reply:
Thanks, this function is indeed applicable to <=3 tags, instead of <3 tags.